### DISCUSSION TITLE
最新稳定版以及cl版无法在真我gt13.1上工作

### DISCUSSION THREAD

**Discussion by 4AMxxxlu** [LSPosed/LSPosed#2585] [General]
升级了最新F12系统版本13.1lsp不工作!图片来自酷安评论区[Screenshot_2023-06-28-10-53-41-27_fc704e6b13c4fb26bf5e411f75da84f2.jpg](https://github.com/LSPosed/LSPosed/assets/75058450/7ac289a2-9249-4062-9c22-a7a5f3e3d86a)
![bc9d977897d4f01a9f441d42194aa729.jpeg](https://github.com/LSPosed/LSPosed/assets/75058450/c43c185a-81d7-4bce-b3cc-bfc036219d34)

![609767368262f805c6aca808e785deeb.jpeg](https://github.com/LSPosed/LSPosed/assets/75058450/c53176af-1d83-4955-bec0-eba0ad8a1c7b)

**Comment by 4AMxxxlu** [comment]
![Screenshot_2023-06-28-10-53-41-27_fc704e6b13c4fb26bf5e411f75da84f2.jpg](https://github.com/LSPosed/LSPosed/assets/75058450/17e92563-ab7c-43ab-87c1-1e9d067aa79d)

**Comment by yujincheng08** [comment]
那得等我有钱买台同款机型才能修了:)
